react/src/App.jsx

```diff
@@ -0,0 +1,302 @@
+import { useState, useCallback, useEffect } from "react";
+
+function App() {
+  const [position, setPosition] = useState("bottom-right");
+
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const [selectedPrivacyOption, setSelectedPrivacyOption] = useState("disable");
+  const [selectedFullScreenOption, setSelectedFullScreenOption] =
+    useState("exitFullScreen");
+
+  const [selectedLauncherOption, setSelectedLauncherOption] =
+    useState("showLauncher");
+
+  const [unreadCount, setUnreadCount] = useState(0);
+  useState(false);
+
+  const [message, setMessage] = useState("");
+  const [messageEventListenerAdded, setmessageEventListenerAdded] =
+    useState(false);
+
+  const updateHelpshiftConfig = () => {
+    window.Helpshift("updateHelpshiftConfig");
+  };
+
+  useEffect(() => {
+    const newInterval = setInterval(() => {
+      setMessage("");
+    }, 3000);
+
+    return () => clearInterval(newInterval);
+  }, [message]);
```

Comment:
Please explain why you are adding this.
Do we need to clear the message after 3s? Why not just keep it?

Reply:
Ok,
- Basically i was adding this because when we have added the event and then we login or logout so ideally the message that is being shown should be empty string but it's remaining the same 
- that is why i have added the set Interval
- But yes, we can always show the message and when the login/logout happen i will set the message as empty string

Reply:
Ah, got it!
Please update it to be cleared on Login/Logout.